### PR TITLE
v1.25.0 - creation of `.o-formToggle` with images

### PR DESCRIPTION
**Pull request by sonnyprince**
- creation of `.o-formToggle` with images
- remove of unused styles from formToggle.

screen shot 2019-01-04 at 12 15 05

- [x] This PR has been checked with regard to our brand guidelines
- [x] This code has been checked with regard to our accessibility standards

## Browsers Tested

- [x] Chrome
- [x] Edge
- [x] Internet Explorer 11
- [x] Mobile

### CODE REVIEW

File: src/scss/objects/_form-toggle.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * =================================
  * Styles a button on top of a checkbox, to toggle inputs to a form
  *
- * Example: Search page filters (sidebat)
```
Comment (DamianMullins) [inline src/scss/objects/_form-toggle.scss]: 🦇

**Comment by DamianMullins** [comment]
Please tell me those are the images we're going to use in production? 😄

**Comment by sonnyprince** [comment]
@DamianMullins If we don't we're missing a trick..

File: src/scss/objects/_form-toggle.scss [hunk 2]
```diff
@@ -160,27 +139,40 @@ $formToggle-checked-text            : $green;
...
+                }
+            }
         }
 
-        .o-formToggle--largeAboveMid {
-            @include media('>=mid') {
-                float: left;
-                margin: 0 0 spacing();
-                @include formToggle-large();
+            .o-formToggle--withImage-container {
```
Comment (ashleynolan) [inline src/scss/objects/_form-toggle.scss]: I'd be tempted to structure this as:

```
.o-formToggle--withImage {
   & .o-formToggle-container {
   }
}
```

Just makes it a bit more future proof if other modifiers need to hook onto and use `o-formToggle-container`

Comment (sonnyprince) [reply src/scss/objects/_form-toggle.scss]: yeah, that makes sense - will do.

**Comment by ashleynolan** [comment]
Does this still need reviewing/merging? Will need a date/version bump if so (and the do not merge to be removed)

**Comment by ashleynolan** [comment]
@sonnyprince Any news on this PR (just trying to make sure we don't have PRs open indefinitely)

**Pull request closed without merging**